runner/internal/shim/docker.go

```diff
@@ -151,6 +151,14 @@ func (d *DockerRunner) restoreStateFromContainers(ctx context.Context) error {
 						gpuIDs = append(gpuIDs, device.PathOnHost)
 					}
 				}
+			case host.GpuVendorTenstorrent:
```

Comment:
@un-def not sure if this code is current and how to test it properly

Reply:
1. Start a run
2. Restart the shim: `systemctl restart dstack-shim.service`
3. Check `shim.log` for the following line: `locked GPU(s) due to running task`

Reply:
Yes, I see
> time=2025-04-30T11:05:26.154875+02:00 level=debug msg=locked GPU(s) due to running task task=a9c04738-e690-425f-9566-4fa1f4943abc gpus=[0]

Does it mean everything works, and we can merge?